### DISCUSSION TITLE
Redirect loop for NGINX proxy.

### DISCUSSION THREAD

**Discussion by artificialai223** [plan-player-analytics/Plan#2397] [Archive]
(I know it shows the IP, Does not matter.) https://www.toptal.com/developers/hastebin/inuqeriquz.nginx - NGINX

https://www.toptal.com/developers/hastebin/ocodopofew.yaml - Plan

![](https://i.imgur.com/R9rvY8u.png)

**Comment by AuroraLS3** [comment]
Did you reload Plan after you set the alternative IP?

**Comment by artificialai223** [comment]
Yes, Full server reboot.

**Comment by artificialai223** [comment]
Cloudflare full ssl mode is required.
